python/Generators/Generators.py

```diff
@@ -1,3 +1,5 @@
+import importlib
+
 from . import Sherpa
```

Comment:
If you dynamically import them, you probably don't have to statically do it here any longer.

Reply:
Thx, done